client/app/styles/onboarding.styl

```diff
@@ -87,12 +87,18 @@ form
             top:        0
             right:      0
 
+        button.icon
+            background-size: 1em 1em
+            margin: .55em 0
+
+            span
+                display: none
 
         .hide-password
-            background: red
+            background: aqua url(/app/assets/sprites/hide-eye-icon.svg)
```

Comment:
@m4dz : this actually doesnt work; can you tell me which path I can use? I looked at webpackconfig but I didn't understand how to do it. Thx